plugins/polio/models.py

```diff
@@ -1054,6 +1054,65 @@ class VaccineArrivalReport(SoftDeletableModel):
     objects = DefaultSoftDeletableManager()
 
 
+class VaccineStock(models.Model):
+    account = models.ForeignKey("iaso.account", on_delete=models.CASCADE, related_name="vaccine_stocks")
+    country = models.ForeignKey(
+        "iaso.orgunit",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="vaccine_stocks",
+        help_text="Unique (Country, Vaccine) pair",
+    )
+    vaccine = models.CharField(max_length=5, choices=VACCINES)
+
+    class Meta:
+        unique_together = ("country", "vaccine")
+
+
+# Form A
```

Comment:
What does "Form A" mean? Is it a business-related term?
It's a bit meaningless out of context.
I suggest removing it or explaining with a bit more details in the docstring?